packages/api/src/api/user.ts

```
@@ -0,0 +1,183 @@
+   * @request GET:/user/{username}
+   */
+
+  async getUserByName<T = User>() {
+    try {
+      const method = "get";
+      const endpoint = "/user/${username}";
+      const options = undefined;
+
+      return await apiInstance[method]<T>(endpoint, options);
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)